sw/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ project(demo_system_sw)
 set(COMMON_DIR "${CMAKE_CURRENT_LIST_DIR}/common")
 set(LINKER_SCRIPT "${COMMON_DIR}/link.ld")
 set(CMAKE_C_COMPILER riscv32-unknown-elf-gcc)
-set(CMAKE_EXE_LINKER_FLAGS "-nostdlib -nostartfiles -T${LINKER_SCRIPT}")
+set(CMAKE_EXE_LINKER_FLAGS "-nostartfiles -T${LINKER_SCRIPT}")
```

Comment:
Is there something we're using from stdlib?

Reply:
The sw implementation of floating point.